01_prepare_host.sh

```diff
@@ -14,10 +14,20 @@ else
   source centos_install_requirements.sh
 fi
 
-if ! command -v minikube 2>/dev/null ; then
-    curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
-    chmod +x minikube
-    sudo mv minikube /usr/local/bin/.
+if [ "${EPHEMERAL_CLUSTER}" == "minikube" ]; then
+  if ! command -v minikube 2>/dev/null ; then
+      curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+      chmod +x minikube
+      sudo mv minikube /usr/local/bin/.
+  fi
+fi
```

Comment:
this will make configure_minikube and init_minikube function on line 123 and 124 fail. They need to be conditionnal too.